src/NavItem.ts

```diff
@@ -0,0 +1,23 @@
+class NavItem extends HTMLElement {
+  static NAV_ITEM_NOT_FOUND_ERR = 'Nav Item Not Found';
+  static HTML = `
+  <li>
```

Comment:
The HTML here is invalid. Please do some research on this before heading into development. Please refer to this [link](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements#using_the_lifecycle_callbacks) to learn ** attributeChangedCallback**. The HTML will be something like this:

```html
<my-element data-name="John" data-age="30"></my-element>
```